homework3/semi-uber/Server/api/handlers/trucksHandlers/trucksPatchHandler.js

```diff
@@ -0,0 +1,33 @@
+const log4js = require("log4js");
+const logger = log4js.getLogger();
+const errorHandler = require("../error.handler");
+const { Truck } = require("../../Schemas/truck.schema");
+
+const trucksPatchHandler = async (req, res) => {
+  const { userID, truckID } = req.body;
+
+  try {
```

Comment:
Please change this logic as we have discussed:
Find drivers assigned truck, set `assigned_to` to `null`. And then assign the new truck to the driver

Reply:
Done